Automattic/wp-calypso
# Code Clean Up, Remove deprecated Google Analytics (GA3) References

Related to # https://github.com/Automattic/wp-calypso/pull/90481 https://github.com/Automattic/martech/issues/2318

## Proposed Changes
Google Universal Analytics (GA3 UA) has been deprecated and we have updated to Google Analytics 4.
This PR Continues the code clean up of removing unused GA3 UA code started in https://github.com/Automattic/wp-calypso/pull/90481

1. Remove deprecated  GA Enhanced Eccommerce
2. Remove unit tests under test/google-analytics
3. Removing deprecated ad-tracking/google-analytics
4. Refactor code to refer to ad-tracking/google-analytics-4 instead of ad-tracking/google-analytics


## Testing Instructions

1. Check out the branch
2. Allow ad-tracking and the GDPR banner in development.json
3. Open Calypso as a logged-in user.
4. You should see GA4 pixel being loaded, and the page_view and user_engagement events should be sent. 
5. Visit another page and the page_view event should be sent.
6. Sandbox the API and purchas a plan. 
7. You should see GA4 conversions being sent.
